psydac/cmd/accelerate.py

```diff
@@ -0,0 +1,10 @@
+def main( language='fortran'):
+    import os
+    psydac_path=os.path.dirname(os.path.abspath(__file__))
+    print(psydac_path)
+    os.system('pyccel ' + psydac_path + '/../linalg/kernels.py --language '+ language)
+    os.system('pyccel ' + psydac_path + '/../api/ast/linalg_kernels.py --language '+ language)
+    os.system('pyccel ' + psydac_path + '/../core/kernels.py --language '+ language)
+    os.system('pyccel ' + psydac_path + '/../core/bsplines_pyccel.py --language '+ language)
+    return
```

Comment:
Are you sure that the `language` parameter is read from the terminal?

Reply:
@yguclu I change the method by creating an environment variable named PYCCEL_LANGUAGE. The method `pyccel-accelerate` will read the variable and checked if it is a supported language. If the variable doesn't exist, it will pyccelise the files with fortran

Reply:
This issue is outdated